### ISSUE TITLE
Problems About testset and trainset link

### ISSUE THREAD

**Issue by louzq16** [SeasonDepth/SeasonDepth#4]
I have downloaded your released testset and trainset. It seems that the testset link contains training data and the trainset link contains test data, which has confused me a lot.

**Comment by yangbaoquan** [comment]
Thanks a lot for using SeansonDepth.
Actually, you can have access to the validation set and training set from our [official web page](https://seasondepth.github.io/). You can also simply click the following link: [validation set](https://figshare.com/articles/dataset/SeasonDepth_Cross-Season_Monocular_Depth_Prediction_Dataset/14731323), [training set](https://figshare.com/articles/dataset/SeasonDepth_Cross-Season_Monocular_Depth_Prediction_Training_Dataset/16442025)
The training set is consist of slice4, 5 and 6, while the validation set is consist of slice 2, 3, 7 and 8, which can be found in [README.md](https://github.com/SeasonDepth/SeasonDepth#1-dataset-preparation) and [dataset_info/README.md](https://github.com/SeasonDepth/SeasonDepth/tree/master/dataset_info#seasondepth-test-and-training-set-for-benchmark).
Hope that can help you.

**Comment by louzq16** [comment]
Thanks a lot for your reply
